app/static/js/createEvents.js

```diff
@@ -103,20 +154,78 @@ $(document).ready(function () {
   $("#saveEvent").on('submit', function (event) {
     $(this).find("input[type=submit]").prop("disabled", true);
   });
+  
+  var modalOpenedByEditButton = false;
 
-  $("#checkIsRecurring").click(function () {
-    var recurringStatus = $("input[name='isRecurring']:checked").val();
+  $("#checkIsRecurring, #checkIsMultipleOffering, #edit_modal").click(function(event) { //#checkIsRecurring, #checkIsMultipleOffering are attributes for the toggle buttons on create event page
+    if(!(document.getElementById('inputEventName').value === '')){
+      document.getElementById('eventName').value = document.getElementById('inputEventName').value; //keeps main page event name for multiple event modal
+    }
+    var recurringStatus = $("input[id='checkIsRecurring']:checked").val(); //this line function is to retrive ON when its toggle for recurring event on createEvent.html line 158
+    var multipleOfferingStatus = $("input[id='checkIsMultipleOffering']:checked").val();// this line function is to retrive ON when toggle for multiple offering event button createEvent.html line 160
```

Comment:
Let's put all of these end-of-line comments before the lines instead of at the end. It's easier to follow.

Reply:
Also, these selectors are complicated. we don't need to use an attribute selector if we know an id, we can just use the id. That is, instead of `$("input[id='checkIsRecurring']:checked")`, use `$("#checkIsRecurring:checked")`.

Or maybe use booleans everywhere instead of 'on' or '', with `$("#checkIsRecurring").is(":checked")`.

Reply:
replaced current way of checking if these are toggled to now be booleans as suggested.